platform/pulpcore/app/views/status.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from pulpcore.app.serializers.task import WorkerSerializer
+import status
+
+
+class StatusView(APIView):
+    """
+    Returns status information about the application
+    """
+    authentication_classes = []
+    permission_classes = []
+
+    def get(self, request, format=None):
+        """
+        Returns app information including the version, known workers, database connection status,
+        and messaging connection status
```

Comment:
Obvious as it may be, I'm a fan of documenting the parameters, even if that means copy-pasting from the parent class. Since parent classes can change, that at least captures what your expectations were at the time of writing the subclass. It also helps the reader immensely.